storagenode/storagenodedb/consoledb.go

```diff
@@ -26,17 +26,37 @@ func (db *InfoDB) Console() console.DB { return &consoledb{db} }
 // Console returns console.DB
 func (db *DB) Console() console.DB { return db.info.Console() }
 
-// GetSatelliteIDs returns list of satelliteIDs that storagenode has interacted with
+// Satellites returns consoledb as console.Satellites
+func (db *consoledb) Satellites() console.Satellites {
+	return db
+}
+
+// Bandwidth returns consoledb as console.Bandwidth
+func (db *consoledb) Bandwidth() console.Bandwidth {
+	return db
+}
+
+// DiskSpaceUsages returns consoledb as console.DiskSpaceUsages
+func (db *consoledb) DiskSpaceUsages() console.DiskSpaceUsages {
+	return &diskSpaceUsage{InfoDB: db.InfoDB}
+}
+
+// Stats returns nodeStats as console.Stats
+func (db *consoledb) Stats() console.Stats {
+	return &nodeStats{InfoDB: db.InfoDB}
+}
+
+// GetIDs returns list of satelliteIDs that storagenode has interacted with
 // at least once
-func (db *consoledb) GetSatelliteIDs(ctx context.Context, from, to time.Time) (_ storj.NodeIDList, err error) {
+func (db *consoledb) GetIDs(ctx context.Context, from, to time.Time) (_ storj.NodeIDList, err error) {
 	defer mon.Task()(&ctx)(&err)
 
 	var satellites storj.NodeIDList
 
-	rows, err := db.db.QueryContext(ctx, db.Rebind(`
-		SELECT DISTINCT satellite_id
+	rows, err := db.db.QueryContext(ctx,
+		`SELECT DISTINCT satellite_id
 		FROM bandwidth_usage
-		WHERE ? <= created_at AND created_at <= ?`), from.UTC(), to.UTC())
+		WHERE ? <= created_at AND created_at <= ?`, from.UTC(), to.UTC())
```

Comment:
I dont think we need that here. Query all and ensure we get stats for all of them.
Besides that the boundries are both selecting if the time is lower or equal then X. One needs to be greater or equal.

Reply:
`before <= created_at <= after`. After is greater and before is lower, no? Or I didn't understand what you mean

Reply:
Also If I want monthly stats, do I still need all of the satellites on Dashboard?

Reply:
Oh, my fault. I misread the order.
The other question, why to we only focus on satellites that you have interacted with? If you just store data and dont get a single audit, upload or download, the satellite would be missing, I am voting for querying all satellites here.

Reply:
Done, the question was, show all or only those that were active during suggested time